db/data/20250220124403_policy_options_provided_lup_to_tri.rb

```diff
@@ -0,0 +1,13 @@
+# Run me with `rails runner db/data/20250220124403_policy_options_provided_lup_to_tri.rb`
+
+r = ActiveRecord::Base.connection.execute("SELECT id from (SELECT id, jsonb_path_query(policy_options_provided, '$.policy') AS options FROM \"claims\") as inner_table WHERE options = '\"LevellingUpPremiumPayments\"'")
+
+ids = r.map { |e| e["id"] }
```

Comment:
I _think_ we could do
```
ids = Claim.where(%{policy_options_provided @> '[{"policy": "LevellingUpPremiumPayments"}]'}).pluck(:id)
```
but not a blocker for a data migration 👍

Reply:
i always have to look up json syntax, don't use it enough to memorise, and ends up being different every time